examples/apps/ai-collab/src/components/UserProfilePhoto.tsx

```diff
@@ -0,0 +1,125 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
```

Comment:
This file should have the `"use client";` directive at the top

Reply:
Right. Updated